pulsar-client/src/main/java/org/apache/pulsar/client/util/ExceptionHandler.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pulsar.client.util;
+
+
+public class ExceptionHandler {
+
+    public static void handleInterruptedException(Throwable throwable){
```

Comment:
can you please add a javadoc that explains how to use this method and why we should use it ?

Reply:
@eolivelli Okay ! I will add javadoc to explain how and why to use it! 
But this class only provide static Util method ，should we worry about it being instantiated?
And why move it to pulsar-common? I mean whether there are somewhere that cathing InterruptedException and should reset thread interruted flag.